frontend/demo/src/pages/Inbox/ConversationListItem/index.module.scss

```diff
@@ -0,0 +1,180 @@
+@import '../../../assets/scss/colors.scss';
+@import '../../../assets/scss/fonts.scss';
+
+.clickableListItem {
+  padding: 0px 8px;
+
+  > a {
+    text-decoration: none;
+  }
+}
+
+.containerListItem {
+  padding: 16px 8px 0 16px;
+  display: flex;
+  flex-direction: row;
+  border: 1px solid rgba(0, 0, 0, 0);
+}
+
+.containerListItem:hover {
+  svg {
```

Comment:
The svg part can be skipped here I think. Nesting closures in sass always means "any descendant"